### ISSUE TITLE
Unable to extract

### ISSUE THREAD

**Issue by Eddievanoel** [ytdl-org/youtube-dl#30604]
Thies@Laptop:~
$ youtube-dl --version
2021.12.17
Thies@Laptop:~
$ youtube-dl https://tele5.de/mediathek/below-verschollen-im-schnee/
[Tele5] below-verschollen-im-schnee: Downloading webpage
ERROR: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

Can you help?

Thx

######################################################################
  WARNING!
  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN ISSUE CLOSED AS INCOMPLETE
######################################################################

-->

## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
PASTE VERBOSE LOG HERE
```Thies@Laptop:~
$ youtube-dl --version
2021.12.17
Thies@Laptop:~
$ youtube-dl https://tele5.de/mediathek/below-verschollen-im-schnee/
[Tele5] below-verschollen-im-schnee: Downloading webpage
ERROR: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.



## Description
Thies@Laptop:~
$ youtube-dl --version
2021.12.17
Thies@Laptop:~
$ youtube-dl https://tele5.de/mediathek/below-verschollen-im-schnee/
[Tele5] below-verschollen-im-schnee: Downloading webpage
ERROR: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

<!--
Provide an explanation of your issue in an arbitrary form. Please make sure the description is worded well enough to be understood, see https://github.com/ytdl-org/youtube-dl#is-the-description-of-the-issue-itself-sufficient. Provide any additional information, suggested solution and as much context and examples as possible.
If work on your issue requires account credentials please provide them or explain how one can obtain them.
-->

WRITE DESCRIPTION HERE
Thies@Laptop:~
$ youtube-dl 

- [ ]   ##--version

2021.12.17
Thies@Laptop:~
$ youtube-dl https://tele5.de/mediathek/below-verschollen-im-schnee/
[Tele5] below-verschollen-im-schnee: Downloading webpage
ERROR: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

**Comment by 89z** [comment]
Regional lockout

> Error Code: access.denied.geoblocked

**Comment by dirkf** [comment]
Also, fixed in PR #30532 that was merged last night.
